### ISSUE TITLE
Useless information on Loader API's this.rootContext

### ISSUE THREAD

**Issue by soletan** [webpack/webpack.js.org#2579]
**Feature to document**

[Docs on Loader API](https://webpack.js.org/api/loaders/#this-rootcontext) explain `this.rootContext` like this:

> Starting with webpack 4, the formerly this.options.context is provided as this.rootContext.

That's just some background information on _why_ this API property is exposed. It lacks describing _what_ this property is good for (though it might be guessable for some people). Maybe it would help checking out [information on this.options](https://webpack.js.org/api/loaders/#this-options) for clarification, but:

> The options property has been deprecated in webpack 3 and removed in webpack 4. 

That's all. So there is no useful information except some background story to be told on holiday parties.

**Author(s)**

_n/a_

**Additional information**

_n/a_

**Comment by milottit** [comment]
I agree. Somebody new in webpack, not knowing webpack 3, cannot guess what means this.rootContext. An explanation is required.
